success-criteria.md

```diff
@@ -443,39 +425,29 @@ stakeholder group.
 -   The publisher ad-funded business model is supported by addressing marketers
     needs and wants.
 
-    -   [Impacting these marketer interests, reduces publishers' revenue earned](https://services.google.com/fh/files/misc/disabling_third-party_cookies_publisher_revenue.pdf)
+    -   The capacity to positively match type and quality of publishers' content with advertising content
```

Comment:
Lacks clarity. Proposed to strike the first, as it appears to be assuming that marketers' interests can only be supported by 3p cookies, instead of a variety of measures.